dll/win32/shell32/CDefView.cpp

```diff
@@ -1426,6 +1426,15 @@ LRESULT CDefView::OnContextMenu(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &b
     if (!m_hContextMenu) 
         return E_FAIL;
 
+    LV_HITTESTINFO hittest = { { GET_X_LPARAM(lParam), GET_Y_LPARAM(lParam) } };
+    ScreenToClient(&hittest.pt);
+    m_ListView.HitTest(&hittest);
+    if ((hittest.flags & LVHT_ONITEM) &&
+        m_ListView.GetItemState(hittest.iItem, LVIS_SELECTED) != LVIS_SELECTED)
```

Comment:
At first glance it seems correct, but can you please verify that it still works correct with the menu key on the keyboard?

It's this key on a european keyboard:

![afbeelding](https://user-images.githubusercontent.com/610685/69225045-e6d8b200-0b7d-11ea-90e7-2819adbe2a2d.png)

Reply:
The menu key returns `GET_X_LPARAM(lParam) == -1, GET_Y_LPARAM(lParam) == -1`.
Selection is wrong.

Reply:
FIXED.

Reply:
The menu key should not change the selection.